<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDialog</class>
 <widget class="QDialog" name="OrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>填写订单信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_ticket">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>460</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>票务信息</string>
   </property>
   <widget class="QLabel" name="label_type_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>类型：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ticketNo_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>编号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ticketNo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_route_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>路线：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_route">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>350</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_departure_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>出发：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_departure">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_arrival_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>到达：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_arrival">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_price_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>单价：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_company_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>公司：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_company">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_available_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>可用：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_available">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_passenger">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>460</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>乘客信息</string>
   </property>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>28</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>请输入乘客姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>身份证：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>63</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>请输入身份证号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>电话：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>98</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>请输入联系电话</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>数量：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_count">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>28</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>65</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总价：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>65</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>color: red; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>0.00 元</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认订票</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
